tsconfig.base.json

```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "noImplicitAny": true,
     "sourceMap": true,
+    "allowSyntheticDefaultImports": true,
```

Comment:
We need this to be able to do `import pick from 'lodash.pick'`

https://www.typescriptlang.org/tsconfig#allowSyntheticDefaultImports